assembl/static2/js/app/components/debate/thread/post.jsx

```diff
@@ -194,7 +205,19 @@ class Post extends React.PureComponent {
     };
     return (
       <div className="posts" id={id}>
-        <Nuggets extracts={extracts} postId={id} />
+        <Nuggets
+          extracts={extracts}
+          postId={id}
+          nuggetsManager={nuggetsManager}
+          completeLevel={(fullLevel
+            ? [
+              rowIndex,
+              ...fullLevel.split('-').map((string) => {
+                return Number(string);
+              })
+            ]
+            : [rowIndex]).join('-')}
```

Comment:
This entire if block is next to unreadable for a developer not very familiar with this part of the code. For the sake of clarity, can it be written outside of the Component creation as a variable and cleaned up?